### PR TITLE
feat: Add DirectX 12 command

### PR DESCRIPTION
**Pull request by Earthsam12**
## Description

Since the release of Sim Update 7 and the addition of DirectX 12 support, the server has received many questions regarding it.
This PR adds a simple explanation command of the current state of MSFS, FBW and DX12 via ".dx12".

## Test Results

image

## Discord Username

Earthsam12#5545

### CODE REVIEW

**Comment by oim1** [comment]
I feel like this would be better under the 'support' category instead of general. Someone correct me if they feel like general is appropriate.

**Comment by Earthsam12** [comment]
> I feel like this would be better under the 'support' category instead of general. Someone correct me if they feel like general is appropriate.

Ok I can do this later today.

Also, do you think that I should add clarification about how DX12 doesn't have any performance increase? I've seen more of those questions in the server too.

**Comment by benw202** [comment]
@Earthsam12 Performance is sometimes a bit of a grey area as users have a whole host of different hardware/configurations etc. If you feel it is mandatory/people are asking a lot, you could say your results may vary bla bla, but maybe not give a solid 'no' to it?

**Comment by Earthsam12** [comment]
> @Earthsam12 Performance is sometimes a bit of a grey area as users have a whole host of different hardware/configurations etc. If you feel it is mandatory/people are asking a lot, you could say your results may vary bla bla, but maybe not give a solid 'no' to it?

I think I'll just leave it as is then, since the main point of the command is about the FBW addon with DX12, rather than the sim in general. 
I'll commit to move the command to src/commands/support in just a moment

**Comment by tracernz** [comment]
Instead of saying there's an issue with it, maybe something like:
> Microsoft is offering a preview of DirectX 12 support from Sim Update 7 (November 2021). FlyByWire cannot offer support for users of the DirectX 12 preview at this time. If you do decide to experiment, please ensure you test with DirectX 11 prior to reporting any issues.

**Comment by Earthsam12** [comment]
> Instead of saying there's an issue with it, maybe something like:
> > Microsoft is offering a preview of DirectX 12 support from Sim Update 7 (November 2021). FlyByWire cannot offer support for users of the DirectX 12 preview at this time. If you do decide to experiment, please ensure you test with DirectX 11 prior to reporting any issues.

The issue with DX12 is the stuttering with glass cockpit tech in the FBW, so it would be counter productive to advise users that they can experiment with it as they would likely end up coming back asking for support and wasting time, defeating the purpose of the command.

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/commands/support/directx12.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { makeEmbed } from '../../lib/embed';
+
+export const directx12: CommandDefinition = {
+    name: 'dx12',
+    description: 'Explaination for the current state of MSFS DirectX12 support.',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'MSFS | DirectX 12',
+        description: 'Use of DX11 is still recommended for best performance due to an issue with some glass cockpit technologies in the DX12 version of the simulator. '
+                + 'This is not a FBW issue and we cannot address it for now - please keep in mind DX12 is officially a "beta feature" and thus should not be used unless you know what you are doing. '
```
[the remark this answers is not in the document]
Comment (benw202) [reply src/commands/support/directx12.ts:12]: Could be useful to say that we won't provide support?

**Review by benw202 — APPROVED** [review]

**Pull request merged**